docs/source/tutorials/e2e_flow.rst

```diff
@@ -116,31 +134,76 @@ tokenizer. Let's do this using the overrides in the tune CLI while starting trai
 
 .. code-block:: bash
 
-    tune run lora_finetune_single_device \
-    --config llama2/7B_lora_single_device \
-    checkpointer.checkpoint_dir=<checkpoint_dir> \
-    tokenizer.path=<checkpoint_dir>/tokenizer.model \
-    checkpointer.output_dir=<checkpoint_dir>
+    tune run lora_finetune_single_device --config llama3_2/3B_lora_single_device
 
 
-Once training is complete, you'll see the following in the logs.
+Preparing your artifacts for inference
+--------------------------------------
 
-.. code-block:: bash
-
-    [_checkpointer.py:473] Model checkpoint of size 9.98 GB saved to <checkpoint_dir>/hf_model_0001_0.pt
+Congrats for getting this far! You have loaded your weights, trained your model, now it's time to visualize
+the outputs. A simple way of doing this is by running `tree -a path/to/outputdir`, which should show something like the tree below.
+There are 4 types of folders:
```

Comment:
I think there are only 3 types of folders 😃

Reply:
Sorry, bad with numbers